### ISSUE TITLE
Scuba's Ep3 Playthrough

### ISSUE THREAD

**Issue by PixelScuba** [Realm667/WolfenDoom#322]
**General WTF**

- [x] Do the Eisenman and other story-related items have to appear in your inventory? They make it hard to quickly cycle to thing you need.
- [x] Still unsure about having Nazi medics you have to shoot wearing red crosses - shooting noncombatants is the very reason the red cross protects it's symbol - it never wants to be seen as party to any side. What about white with a red swastika instead of a cross? Then you know they’re Nazis and it implies “healer”
- [x] The last slashing frame of the sword looks stupid (in both directions)… should just be a continuation of the previous frame _[assigned to @Ozymandias81 @DoomJedi]_
- [x] Do all the female characters in the credits need to have ridiculously large breasts? Seems tacky. 
> Regarding the big breasts, we have discussed this internally and we'd like to keep it, but the tortured naked woman will be removed.  --T667
- [x] (Not a bug many will encounter and BoA may not even be optimized for this) My sister bought me an ultra widescreen monitor for Christmas… if I play on 21:9 all the dialog boxes stretch horizontally.
> Regarding the resolutions, all the GZDoom presets are working flawlessly. As soon as it is larger than 16:9, various things get actaully stretched, not only the dialog boxes. The question here is simply: How much worth (and danger when it comes to introducing new bugs) is the effort that is needed to make the whole mod 21:9 compatible and how many people will benefit from that improvement. In this special case, I'd say it's low priority (sorry for that), especially in this phase of development. I do not know how much work is involved to get this set though. --T667

### **Episode 3**
**Mission01**
- [x] Red door doesn’t open when I have the red key

**Mission02**
- [x] Autosave after the turret segment and before the tanks show up. I was blown up immediately and had to restart the whole mission. Same for arriving at the Allied building outpost.
- [x] Inform the player the Tank has an alternate fire mode for the machine gun? I didn’t realize this until almost the end of the map. Maybe a simple message when you get into the tank?
- [x] Managed to get the tank stuck. https://i.imgur.com/49w3bUP.png https://i.imgur.com/9ObxOLh.png
- [x] (Not high priority) These doors are all over... but the handles are oddly low to the ground. Would make more sense with the handles closer to the middle. https://i.imgur.com/7top9Cj.png

**Mission03**
- [x]  **The topless tortured woman tied up in the basement is exceedingly tacky and out-of-place.** Seriously consider removing this.
- [x] Where do I find the combination to the safe? For that matter... Not necessarily a problem, but are some secrets so hard to find nobody will ever see them? Will anyone ever play Astrostein let alone the Keen maps? Not necessarily a problem, but some secrets are VERY hard to find.

> The safe combination hint sheets are placed in the same map where the safe is. In C3M3_A the hint sheet is in an officer's room in the only optional building you can visit. It is in the northern part near the point where the player's path through the map should turn right, coordinates (3900, 12200, -88). Regarding the secrets' complexity, all the maps you have mentioned are accessed with items that lie openly in non-hidden (but optional) areas of preceding maps, it is just exploration, not secret-hunting. Secrets which are tough to find do exist in BoA, but the reward is resources, so a perceptive player can afford to play a bit more carelessly. --N00b

> I added marks to the safes, as I really want the player to find these. The Keen secrets are so complex, and involved so much work, it would be a shame to miss these. I also added a new text paragraph at the INTERMAP that explains how to open the safes. For the artifacts and files, I already agree with @Username-N00b-is-not-available - it's openly laid out, it only needs exploration.

**Mission04**
- [x] I got really lost after turning on the generators and finding the lamp… I didn’t know where to go for a long time and just wandered the mines. (summon c4 and I had to cheat my way into the mines) Torm---you said I had to push a statue to get the gem... this was so confusing, I never would have figured this out... I think you said you might make this more obvious. Add a quest marker to the compass after you read the note.
- [x] I picked up all the oil for the lantern before I grabbed the lantern. You shouldn’t be able to pick up lantern oil if it’s full.
- [x] “Summon lanternoil” pickup gives error (It says to report it to the devs!)

> T667: OilPickup would be the right item to summon, so this is no bug.

- [x] Autosave before the 2 rolling boulders… I hate dying from a sneaky death and having to replay from my last save.
- [x] I entered a small room with a typewriter, chest, uniform hanging up, and a music change. The door locked behind me and I can’t open it.
- [x] After getting the grappling hook… was completely lost again. I couldn’t even tell I was supposed to go through a hole in the wall above the flames after jumping across the platforms on chains over spikes. Also, jumping across to the opening is so challenging, I thought I wasn’t supposed to be able to do it. In fact… this whole area is confusing. I don’t know what I’m supposed to do or where to go.
- [x] I think I’m supposed to be able to open room 104 in the hotel above the bar… but the door does nothing.
- [x] I can’t open the blue doors to the clothing store despite having the blue key.
- [x] The open sewer grate leading into the opera house is not obvious.

**Mission05**
- [x] I think I’m supposed to have a gas mask while fighting in the city... but I just got sent into the town without one and spent the whole mission choking and slamming medkits to survive.

**Flashback/Amber Room**
- [x] (This is not important, just a thought) Is it possible to run the Amber Room textures through some form of neural upscale? They’re so low resolution compared to everything else… maybe this would make them look comparable to the others?

**Mission06**
- [x] Mengele and Morell used some poison power on me and I just lost health until I died… seems like a LOT of health to loose and completely unfair. I even used “resurrect” and I continued losing health for many revives… Consider drastically reducing the time frame of being poisoned and the amount of damage it does. _[assigned to @Ozymandias81]_

**Mission07**
- [x] I started the mission with a repair kit already… felt odd I had to track down another one. Should you lose specialty inventory items between certain maps?
- [x] I could really use a guide marker on my compass for getting out of the Zeppelin while it’s crashing.
> I am not a big fan of these extremely guided gameplay types (just like in the late Call of Duty games). It is possible to reach the exit while it's crashing in time when it happens for the first time. Even if you need a second try, you will most propably be able to make it then, so I'd simply leave that as is as _gameplay challenge_. --T667

**Mission08 - Wolfenstein**
- [x] I didn’t know the map I purchased in the last level was in my inventory because the inventory icon looked identical to other secret papers I had picked up.
- [x] I'm assuming the ending isn't finished? Hitler was invisible, the spear spawned too high for me to grab, and there was no way for me to get out of the arena_[assigned the Hitler issue to @AFADoomer @Talon1024 @Ozymandias81]_ 
- [x] also a compass marker on where to go once I leave would help.

**Comment by Username-N00b-is-not-available** [comment]
@PixelScuba Thank you for helping us again with your feedback! Some of the issues you mentioned look quite similiar to ones that have been recently reported to be fixed; could you please name the commit and the port version which you have played?
On specific points:

> story-related items litter the inventory bar

Seconding this, the inventory's already full of useful <strike>and useless</strike> items, and you cannot even use the files from the inventory. We can try drawing them outside of the statusbar, similarly to how it was in Stronghold (see the blue floppy disk over the door to the left):
![Screenshot_Doom_20201222_202640](https://user-images.githubusercontent.com/59409713/102916094-0c786f00-4494-11eb-8991-1753e21c4a92.png)

> Nazi medics and the Red Cross

Medics will anyway remain non-combatants in the same way as scientists and mechanics who get under the player's crosshair more often; killing them is a war crime under the international law. So unless our goal here is to please exactly the Red Cross organisation (which might have more important tasks than monitoring fancy new ZDoom mods and games), to change this situation properly we would have to remove them all, or at least change their behaviour to something like the current one of the camp prisoners.

> Red door doesn’t open when I have the red key

Could you please give a screenshot or coordinates of this? Was this on C3M1_A or _B?

> the gem riddle

There is a paper on a wall of this small shed which hints at the location of the gem:

![Screenshot_Doom_20201222_204405](https://user-images.githubusercontent.com/59409713/102917573-8873b680-4496-11eb-820d-d0c6722f620c.png)

> You shouldn’t be able to pick up lantern oil if it’s full

Could you please elaborate on this problem? You cannot pick up the oil if the supply is full and the lamp is not turned on (if it is, the next tic some of the oil will be used up, and the player actor will pick up the oil they are standing on).

> summon LanternOil

OilPickup should work fine, it is the actor which is placed in the map.

> The door locked behind me and I can’t open it.

Very strange, I've been recording C3M0_A just yesterday, got to this room, and was not locked. Maybe this bug was introduced in the last few days?

> Also, jumping across to the opening is so challenging, I thought I wasn’t supposed to be able to do it.

Have you managed to do it? In what way?

> the door does nothing

> can’t open the blue doors

From what I can see there were some serious problems with doors, but for some reason they do not occur to me at all. Have you used a fresh installation of GZDoom? Maybe some settings, or dmflags, or autoloaded separate mods?

> I think I’m supposed to have a gas mask while fighting in the city

Yes, you are :-D It is located here:
![Screenshot_Doom_20201222_205949](https://user-images.githubusercontent.com/59409713/102918813-b4903700-4498-11eb-8ad0-882d2e8a6de3.png)

> multiple final fight problems

All of these problems (invisibility, the spear height, the damaging arena floors) occur quite frequently, but could not be reproduced consistently (e. g. yesterday I noticed that the first two were present, but I could get back to the main area easily), and every time someone fixes them they reappear again after some time. Also there is a compass marker after the fight (a red exclamation mark), maybe it does not appear on some skill settings, or it is the wrong version of BoA, or...

**Comment by PixelScuba** [comment]
I did a pull on the 17th... I can't tell you the version number.

**Comment by Tormentor667** [comment]
@PixelScuba first of all thanks kindly for your feedback. I have a question before I start investigating the problems: Did you use DOOM2.WAD as a base wad or did you launch the Blade of Agony data as IPK3 as it is supposed to be? (stand-alone with GZDoom). This would explain some of the key issues.

**Comment by Talon1024** [comment]
I'm using DOOM2.WAD as the IWAD, and I'm getting no such issues with the keys. Do you have any other mods loaded @PixelScuba ?

**Comment by PixelScuba** [comment]
> @PixelScuba first of all thanks kindly for your feedback. I have a question before I start investigating the problems: Did you use DOOM2.WAD as a base wad or did you launch the Blade of Agony data as IPK3 as it is supposed to be? (stand-alone with GZDoom). This would explain some of the key issues.

I'm using Doom2 as the iwad.

**Comment by Tormentor667** [comment]
Can you try it with boa.ipk3 without doom2.wad and check, if the red key is working properly?

**Comment by PixelScuba** [comment]
Just tried it now... same commit, same gzdooom, but the door now works when running as the iwad.

**Comment by Tormentor667** [comment]
Thanks for testing this. I guess somehow one IWAD/IPK3 overrides the other, even though I do not understand, why this only happens for the red door. Or only for the blue door of the boss store in Mission 3 (C3M0_A) and not for the blue butcher door as well. I mean it's good to know that the IPK3 works, but on the other hand, why doesn't it work when it is launched with the DOOM2.WAD?

**Comment by PixelScuba** [comment]
After you read the note in the shed, it should mark the statue on your map. That would make it less confusing. I also went back and retested areas where I was stuck by a door and they were all related to the iwad/pwad mixup. This seems like a really major problem since there will be a LOT of players who simply drag the file onto gzdoom and run it as a pwad... and will wonder why it doesn't work.

**Comment by Tormentor667** [comment]
I wonder: Why did it work for some locked doors and not for others? Regarding the hint for the statue, can you put that into your first post so we don't miss it?

@Talon1024 @AFADoomer do you think we could add a functionality that makes the game NOT work with any other IWAD is loaded before?

**Comment by Ozymandias81** [comment]
The actual version of BoA work with Doom2 as pk3 or better as ipk3, hopefully you do not tried Freedoom which have internal dehacked that may break keys usage. Also if some gzdoom deafult settings were overridden it may have cause a problem, just assuming like a compatibility flag or something like that. Indeed the version may also be the cause there, I never had such issues when testing but well, I may also be wrong since I am mostly dedicate to debugging stuff here and there in my short amount of free time.

Concerning the "tacky" thing, we'll keep things as they are, Duke Nukem 3d, Blood but also modern fps games are filled with such gory details. Maybe in the future we may think to implement a "parental control" option or something like that, but we are all men of certain age and at least, from me, it is very frustrating when you have to change some details just for the sake of... we decided that way, we liked it during development and we can't satisfy everyone's taste, otherwise the mod won't see the light at all too. In any case thank you very much for your feedbacks, they are always welcomed.

**Comment by AFADoomer** [comment]
One note with the medics...  Could someone add sprites based off of either the crouching prisoner or the surrendering defector that we can replace the medic death state with? That could solve some of the negative perception issue and still remove the actor from active gameplay.

**Comment by PixelScuba** [comment]
It's boorish and completely out-of-place in the context of the game. Answer me this: why does she have to be topless? The woman in the cake... it was juvenile, but as an easter egg that pops out of the cake... tacky sure, but I get it and understand the context. But a woman with her shirt ripped off, exposing her breasts, and being tortured in a dungeon? No, I'm sorry, if you can't provide a rationale beyond "we like it," that's a terrible reason for keeping material in a game that is exceedingly sexist... unless you plan on drawing a guy next to her with his pants pulled down and dick exposed.

Look, I made a game where you beat up stereotyped homosexuals in a bar... looking back, my god, what the fuck was I thinking...  but at the time, I thought it was a clever pastiche on the beat-em-up genre. Instead I just look back and cringe. I'm telling you, this art is a really bad look for BoA.

**Comment by AFADoomer** [comment]
For what it's worth, I'm in agreement with @PixelScuba...

**Comment by DoomJedi** [comment]
I do see the point about medics. 
I think something in this direction could be a good solution, we need to find logical way to take medics out of the "dangerous to player" state into some state logical for the gameplay - without killing them.
It would be a bit weird though if just the medics would surrender. All medics, and just them.
We need to turn them into a state that clearly looks not dangerous anymore - for eternity. Run away or taking some other medic business...picking their medic bag or whatever....donnow.

A do agree about the difference between cake lady and the tortured lady.
I love female nudity in games (my first mod ever was nudity mod, and I'm the one who made all that art for BoA, heh), but that looked out of place even to me. Just because of lack of context the justify that, looks sexist.
But I'm far from "strongly opposing" it, you can keep it if you like it, no big issue for me there.

**Comment by AFADoomer** [comment]
Invulnerable, but dropping their medic bag, then becoming permanently frightened (and disappear when out of line if sight, maybe?) is doable...  would require ABCD/walk sprites with no bag in hand.

**Comment by PixelScuba** [comment]
Too much work, just leave it as is... I mention it however, because I pointed out to Torm earlier that I actually didn't know that what these medics did since I knew, intrinsically, I wasn't supposed to shoot them because they have red crosses. To my surprise, they were healing enemies and I DID have to shoot them. Hence why I thought it might be more effective if they had their white vests with a red nazi swastikas'... it implies "healer" but the symbol means "shoot them."

**Comment by MuadDib90** [comment]
> **PixelScuba said:**
I got really lost after turning on the generators and finding the lamp… I didn’t know where to go for a long time and just wandered the mines. (summon c4 and I had to cheat my way into the mines) Torm---you said I had to push a statue to get the gem... this was so confusing, I never would have figured this out... I think you said you might make this more obvious. Add a quest marker to the compass after you read the note.

After reading this, I got a bad feeling I need to share with all of you.

It is said, that you can complete the game without ever reading the notes, that they are here (more or less) for story purposes. And yes, technically speaking, you can - IF you know, what to do. There is no direct restriction on a map, that prevents you from doing so. But the thing is, without reading those notes (especially on this map), there is a very small chance, player will beat this map. You need that stone from that statue, and you would need to "run around the map" long enough to just "happen to stumble" on a statue to just see, it can move... 

I think you all have now the idea, what I am trying to say...

**Comment by PixelScuba** [comment]
I don't mind "using" the notes... they have giant ! marks over them, so I know they're important... but reading them should highlight my next goal on my compass, otherwise it's baffling. I'd argue the same for other areas where you talk to an NPC who tells you "Sneak into the Operah house through the sewer" and then mark the sewer on the compass.

**Comment by Tormentor667** [comment]
Regarding the exposed woman, I agree with @PixelScuba , @AFADoomer and @DoomJedi. It’s not worth it, it’s indeed sexist, it would fit Duke Nukem 3D but not Blade of Agony so let’s simply remove it.

**Comment by Tormentor667** [comment]
> Too much work, just leave it as is... I mention it however, because I pointed out to Torm earlier that I actually didn't know that what these medics did since I knew, intrinsically, I wasn't supposed to shoot them because they have red crosses. To my surprise, they were healing enemies and I DID have to shoot them. Hence why I thought it might be more effective if they had their white vests with a red nazi swastikas'... it implies "healer" but the symbol means "shoot them."

@PixelScuba and I discussed this issue already via personal message and we will keep the medics as they currently are. On the one hand, he currently has been the only person who pointed this out (which doesn't mean that I do not care). On the other hand, you get to a point where you need to make decisions and that's what I did and do. 

There has been this strange change of "red crosses" to "green crosses" or "pills" in the Bethesda re-releases of Doom (there seems to be a lot of similar movements in Germany when it comes to Grimm's Tales or Jim Knopf, using terms and norms that do not fit to the 2020's anymore, just to give an example) and personally I'd decide something like this according to my gut feeling.

Beyond, we have an explanation about these enemies on our webpage and in the info pages ingame, so I think it's pretty clear what they do and how you need to handle these - we can't be any clearer.

![2020-12-23_210317](https://user-images.githubusercontent.com/10298439/103033227-4dd75000-4562-11eb-8253-cc3a41cb2e2d.jpg)

**Comment by Tormentor667** [comment]
> Made it to the city: Thoughts up to this point—lots of good ideas, but the map layout is unbelievably confusing… lots of little holes in the rock I’m supposed to go through but never saw, hard-to-find items… good ideas that need some better execution to not be frustrating.

Can you ellaborate here? I am willed to improve the corresponding areas but I need some help to know which places you mean and what you think could be improved to make the area less confusing. I mean, the whole map is meant to be some kind of "open world to wander around in", but I don't want to make it frustrating.

**Comment by Tormentor667** [comment]
Did various fixes and improvements today regarding @PixelScuba 's feedback. Thanks for that so far, it's good input!

**Comment by PixelScuba** [comment]
> Can you ellaborate here? I am willed to improve the corresponding areas but I need some help to know which places you mean and what you think could be improved to make the area less confusing. I mean, the whole map is meant to be some kind of "open world to wander around in", but I don't want to make it frustrating.

I deleted that suggestion because it was too vague and some of the confusion was related to getting lost.

**Comment by DoomJedi** [comment]
"he currently has been the only person who pointed this out"
I sure remember quite a few such reports.

**Comment by Tormentor667** [comment]
> "he currently has been the only person who pointed this out"
> I sure remember quite a few such reports.

You mean that people playing the mod didn’t attack the medics because they thought, that medics are not supposed to be hurt or killed?

**Comment by DoomJedi** [comment]
I meant people thinking it's not okay for BoA to have killable red cross medics, for same reasons you mentioned.
They offered to change color and or shape of red cross. I remember we had this discussion before.

**Comment by Tormentor667** [comment]
I still find this question very difficult to answer honestly. Yes, as a modder you loose objectivity. So what are the facts?

- It is clearly stated on the webpage and in the help screens to “shoot the medics first” as they revive your enemies.
- They wear a west with a red cross, where this sign is not connected to the institution but the healing effect, they have on other units (just es the medikits and bandages for example, I mean we don’t use red swastikas on them either)

The latter point is something that I have learned from my childhood experiences when playing Doom, Quake, Blood or Unreal: Items with red crosses improve health. It’s not the Red Cross institution for me, it’s a healing power up. I am pretty sure that I am not the only one who shares this automatism :) 

Long story short: The red cross sign simply implies that the soldiers are healers, but they are still the enemy in this game. Changing the red cross on them means changing the crosses also on medikits, special effects and textures for consistency - and a medikit with a pill or a green cross feels totally wrong to me (I know this is based on my childhood experiences again, so it does not have to be wrong for others :)).

**Comment by PixelScuba** [comment]
![NMDCA1](https://user-images.githubusercontent.com/68576005/103050513-22e8fe00-455a-11eb-82b3-2ceccdf2cdb2.png)

**Comment by DoomJedi** [comment]
...I think you could assume we had enough imagination to understand what you meant without need for actual sprite edit...

Indeed hard to be objective on this.

I still see a big difference between a medikit (that indeed has generic health native gaming interpretation and noone had any issues with it in any mod ever) - and a Person with big red cross on his chest.
Human with red cross - is not like a medikit, it's organization-related uniform and logo. And is very problematic indeed to shoot them to kill.
So even if we change medics - there is no reason whatsoever - whatsoever - to change any other related item in the game artwise.
It's killable human-related uniform that is the issue here.

**Comment by PixelScuba** [comment]
I just like to give visual examples when I can since I'm always worried about language barriers. I took the liberty of modifying all the medic sprites if you want to use them, no credit necessary, I really didn't do much... just a half hour of pixel pushing. https://drive.google.com/file/d/1QoSlxZiSJjtVP_pGbG085HVXQPX1fQsQ/view?usp=sharing

**Comment by Tormentor667** [comment]
What does the rest of the team think about this issue? @Talon1024 @AFADoomer @Username-N00b-is-not-available @DoomJedi @Ozymandias81 @MuadDib90 @doomjuan @madame-rachelle

**Comment by AFADoomer** [comment]
I'm fine with the updated swastika medic sprites.

**Comment by Username-N00b-is-not-available** [comment]
@Tormentor667 Do you mean the medics or the whole issue? If the first, the new medic sprites by @PixelScuba are clearly nicely done, I would like to thank him for the effort, but I tend to not understand the necessity of the replacement.
Even if we look at Blade of Agony as a representation of real-world events (which is a point of view that is questionable in itself, but since we are discussing these matters in such a serious manner for half of this issue, I guess I can also sign up for it), the player will still be killing medics, who are non-combatants (civilians), no matter which symbols will be drawn on their uniforms. By the way, it is widely known that the German Red Cross was disconnected from the main international organization quite a bit during the Nazi times.
I think this is a kind of a big-endian vs. little-endian dispute, changing the symbol would affect neither the ethics nor the gameplay.

**Comment by AFADoomer** [comment]
I'm in the US military. I do not support us having medics wearing a red cross that are almost required to be killed.  The red swastika sprites fix that some, and I'll happily work on some camp-prisoner-style invulnerability and fear behavior for the medic when I get a chance so that they will never be killed.

**Comment by Username-N00b-is-not-available** [comment]
@AFADoomer Actually unkillable medics (but not invulnerable, the player should have ways to stop them at least temporarily) seems to be a good and creative idea, it could spice the gameplay up very nicely (we don't have that many medics in the maps of BoA). Do the resurrected soldiers increase the killcount, BTW?

**Comment by Tormentor667** [comment]
Okay then, thanks for your input on this, guys. If @AFADoomer is willed to adjust the code accordingly, I could imagine making the medics (with red crosses) not die but maybe take cover (knee) if they get hurt. After a while they disappear. That way we can keep the Red Cross, make them unkillable but you still can stop them and the gameplay would change. What do you think about that?

**Comment by Username-N00b-is-not-available** [comment]
@Tormentor667 @AFADoomer We can also consider not making the player shoot the medics at all: a medic would
- work normally when the player can't see him (or, better, when they can't see the player);
- surrender when the player comes close (just conventional surrendering with raising two hands in the air, you can use hands from my C3M5_A actor; please do not make them kneel, this could remind the player of Nemesis);
- then wait until the player goes away + a balance delay of 30 seconds or a minute, after which the medic can go back to resurrecting the soldiers.

This would add a lot of value to the medics as enemies, since now they are just easy targets which are killed by the player upon sight. We will have to balance them though (e. g. make the resurrection sequence way longer, 5 or 7 seconds), so they not become too OP.
I'd personally like to add something similar (but without the resurrection, of course :-) ) to the behaviour of other civilians, scientists and mechanics, a random portion of them could surrender to the player in certain circumstances (if no other Nazi does not see them, for example). Would be nice to have B. J. show some chivalry. I'll see if something comes out of it.

**Comment by Tormentor667** [comment]
But if we go for that approach, do I understand it correctly that you can’t kill/stop the medics at all?

**Comment by AFADoomer** [comment]
I'll make them surrender, then disappear when out of sight of the player for a certain amount of time.

**Comment by Tormentor667** [comment]
Well then, let’s try that. How would we want to change the text accordingly for the info and the webpage?

**Comment by Username-N00b-is-not-available** [comment]
@Tormentor667 What about this? "The Reich has sent medical units to the battlefield, who are able to revive dead soldiers. Do not kill them: if pinned down, they will have no choice but to surrender to you."

**Comment by DoomJedi** [comment]
Nazi medics indeed don't have any weapons, so surrender on threat (they can even wave a white flag if needed) is not as illogical as I initially thought.
Should I start preparing the animation?

**Comment by Tormentor667** [comment]
@DoomJedi I'd suggest you ask @AFADoomer what he has exactly in mind and what additional sprites he needs for that :)

**Comment by DoomJedi** [comment]
Virg, added standard 3-frame Medic Surrender frames.

**Comment by madame-rachelle** [comment]
Considering real life context and even without needing to have read this discussion, I do agree that the medics' red crosses should be removed. While I am not fond of the Red Cross being overly protective of its trademarks (which is protected by international treaty and carries with it some hefty fines from last I read), personnel who wear a red cross are, according to rules of engagement, explicitly supposed to be neutral non-combatants.

Having red cross medics in this mod is a much graver offense than just Doom having random red-cross medkits scattered everywhere.

Even if they do revive enemy soldiers, those are removed from the front lines first, most importantly to be out of danger but explicitly to avoid situations like this. The real life idea of killing red cross medics increases the cost of war because it would make it much too unsafe for neutral people from being able to save lives. In war, both sides appreciate this, even if it does help the enemy.

Having evil medics as is done here is fine - they clearly do not help the player and are meant to be killed - but they should not show any neutral affiliation whatsoever, even if it is unintentional as it is in this case. I have not played this mod extensively, but it is set in a time period where as far as I know, these kinds of standards were followed almost biblically. The sprite edit is an ideal solution to this.

**Comment by AFADoomer** [comment]
The irony is that the sprite is actually reasonably accurate to real-world WWII German medics...  Red cross and all.

**Comment by Tormentor667** [comment]
Well, @DoomJedi did his homework. So we are agreeing on going for @AFADoomer’s suggestion to change their behavior as he proposed, yes?

**Comment by DoomJedi** [comment]
I'm for surrender proposal of Afa, without changing red cross sign. Better with proximity and/or line of sight and not actual need to shoot. But okay either way.

As long as you dont kill them, red cross is fine imho.

**Comment by Tormentor667** [comment]
I agree

**Comment by AFADoomer** [comment]
I've been trying to recreate the key issues that were discussed above that led to the IWAD check being implemented without luck, but I made a minor change in LOCKDEFS in b401250a1fb271b91368d4376cb6145c05bc18b9 that may fix the issue, regardless.

**Comment by Tormentor667** [comment]
Perfect, that's what I ever wanted anyway :-D

**Comment by AFADoomer** [comment]
Does the current implementation of medic surrendering meet everyone's expectations as discussed above?  

Medics will surrender randomly (regardless of how many other Nazis are around) while they are frightened - so, if the player is near them, or if a player bullet tracer or missile passes/explodes near them.  If the player actually shoots them, we pretend the player shot missed, and they surrender immediately.  Medics *can* enter their pain state if damaged by something other than the player, but will still never die (unless they fall from significant height or were to be telefragged somehow).

For all surrendering actors, including the medic, there is a chance of spawning a yellow pool under their feet, and, after surrendering, if no player is in sight for 15 seconds, the actor is removed.

**Comment by Tormentor667** [comment]
I hadn't time yet to try it, currently taking care of fixing issues that had been found from various beta-tests, but from your explanation it sounds quite good. I will take a look as soon as I can, maybe the other devs have some time to check it.

**Comment by DoomJedi** [comment]
Sounds good, about the pool....might be too much, might be not...if made small and rare enough, might work.

**Comment by Tormentor667** [comment]
Fixed all that I was capable of, rest assigned to my teammates :)

**Comment by DoomJedi** [comment]
"The last slashing frame of the sword looks stupid"
SRDGM0/SRDFV0?
Virg, you can just offset previous frame to related sword moving direction and a bit more down
